org.kde.skanlite.json

```diff
@@ -0,0 +1,64 @@
+{
+    "id": "org.kde.skanlite",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "skanlite",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/share/man"
+    ],
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11"
+    ],
+    "modules": [
+        {
+            "name": "sane",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/sane-project/backends/-/archive/RELEASE_1_0_27/backends-RELEASE_1_0_27.tar.gz",
+                    "sha256": "4697959c6bf2efc2af3b24df53bf7c7f1e9086111e93dbada3ec1ed62426d53c"
+                }
+            ]
+        },
+        {
+            "name": "libksane",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.2/src/libksane-21.12.2.tar.xz",
+                    "sha256": "317d4e5948a28180034036268f496279a29ba97ad606522c8ac1ae38b9747ac6"
+                }
+            ]
+        },
+        {
+            "name": "skanlite",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.2/src/skanlite-21.12.2.tar.xz",
+                    "sha256": "15c181b3d155a7222279b52ca7dd83aed2fa02b2edda1cd3c211e8bbfdcf6dbb"
+                },
+                {
+                    "type": "file",
+                    "path": "logo.png",
```

Comment:
it could has well have the proper name from start.

Also, there in no upstream icon?

Reply:
Ok, it is provided by upstream too, but seems like we don't have 128x128 icon.